### PR TITLE
Wolf Pepper Fixing... Again...

### PR DESCRIPTION
**Pull request by Foxwells**
I swear to fuck it was working last time...

Anyway >> More Wolf Pepper fixings. Removed wolf arms tf, but in case anyone has them, the appearance definition is still there. You just can no longer gain them/they don't count toward wolfScore.

If it does't work after this I donno what to say, because it's definitely working for me:

screenshot_146

### CODE REVIEW

File: classes/classes/Items/Mutations.as [hunk 1]
```diff
@@ -1714,386 +1714,366 @@ package classes.Items
...
+			}
+			//MUTATIONZ LEVEL 3: face->eyes
+			if (player.eyeType != EYES_WOLF && player.faceType == FACE_WOLF && rand(4) == 0 && changes < changeLimit) {
+				outputText("\n\nYou feel a sudden surge of pain in your face as your eyes begin to change. You close them and feel something wet slide under your eyelids. You jump in surprise. The feeling's gone, but now the distance is a blurred view, and greens seem to be mixed with yellows.", false);
+				outputText("\n\nYou turn to a nearby reflective surface to investigate. Your eyes have massive amber irises and are dipped into your face, hiding any sign of your sclera. Blackness surrounds them and emphasise the wolfish shape of your face. You blink a few times as you stare at your reflection. <b>You now have wolf eyes!</b> Your peripherals and night vision has probably improved, too.", false);
+				player.eyeType = EYES_WOLF;
+				changes++;
+			}
+			//MISC CRAP
+			if (rand(5) == 0) mutations.updateOvipositionPerk(tfSource);
```
Comment (Stadler76) [inline classes/classes/Items/Mutations.as]: You are already in Mutations.as. No need to reference to it.
```as3
if (rand(5) == 0) updateOvipositionPerk(tfSource);
```
is enough here

File: classes/classes/Items/Mutations.as [hunk 2]
```diff
@@ -1714,386 +1714,366 @@ package classes.Items
...
+					"gray",
+					"dark gray",
+					"light gray",
+					"black",
+					"light brown",
+					"sandy brown",
+					"golden",
+					"silver",
+					"brown",
+					"auburn", ["black", "gray"],
```
Comment (Stadler76) [inline classes/classes/Items/Mutations.as]: `["black", "gray"],` should moved into the next line

Comment (Foxwells) [inline classes/classes/Items/Mutations.as]: Fixed

File: classes/classes/Items/Mutations.as (hunk 1 above)
Comment (Foxwells) [inline classes/classes/Items/Mutations.as]: Also fixed :3

**Pull request merged**